index.js

```diff
@@ -1,10 +1,11 @@
 'use strict'
 
 const CRC = require('crc-32')
+const LRU = require('lru')
 
 class UtilShard {
-  constructor () {
-    this.mem = {}
+  constructor (opts) {
```

Comment:
maybe add an empty object as default value, for some linters maybe complain about missing args

Reply:
done!